### ISSUE TITLE
fully set up Minecraft server image

### ISSUE THREAD

**Issue by huntcsg** [mc-bench/orchestrator#16]
For a couple of reasons, instead of using the existing Minecraft docker image, we should build off of it and actually get the entire server setup, stop it, and commit the filesystem changes.

this should reduce startup time, but more importantly it will ensure a reliable Minecraft runtime and build process.

For license reasons I'm under the impression we cannot distribute this image, but I see no reason we can't build and save this image in a private image repo.

Bottom line: the less runtime phase building we have going on, the better, for performance and for reliability.
